cli/project/files.go

```diff
@@ -47,18 +47,30 @@ func GetStateboardPidFile(projectCtx *ProjectCtx) string {
 
 func GetInstanceConsoleSock(projectCtx *ProjectCtx, instanceName string) string {
```

Comment:
Do you check that its length is less than 108 symbols?

Reply:
In the old cli we checked this only for notify socket that is used directly by `cartridge-cli`. For console sock this check will be performed by `cartridge`. It looks like a good approach.